pages/guides/developing/viewer-engagement.en-US.mdx

```diff
@@ -8,23 +8,61 @@ import { Tab, Tabs } from 'nextra-theme-docs';
 
 # Explore Viewer Engagement
 
+Livepeer offers detailed information on viewer behavior and playback quality on your platform. The API includes engagement metrics such as view counts and watch time, as well as performance metrics such as error rate, time to first frame, rebuffer ratio, and exit-before-starts across a variety of dimensions.
+
+### Performance Metrics
+
+This API offers several key performance metrics to help you understand your platforms performance:
+
+- Error Rate: Percentage of views that encountered an error.
+- Time to First Frame (TTFF): Measures the time in milliseconds it takes between the player requesting video and the first frame being ready to play.
+- Rebuffer Ratio: Percentage of time a viewer spends experiencing rebuffering during playback.
+- Exit Before Starts: Percentage of views that are abandoned before the video begins to play, potentially indicating issues with playback.
+
+### Usage Metrics
```

Comment:
nit: can we move Usage Metrics to the top? Most users we've talked to are more concerned about engagement, so we'll want that "above the fold"